exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
```

Comment:
Any particular reason to go with `subject`? It seems less intuitive to me than `school` without even considering the class is named `School`. I think it could be distracting to the user.